docs/UserGuide.md

```diff
@@ -449,10 +615,30 @@ Exits the program.
 Format: `exit`
 
 #### 4.3 Saving the data :
+All tCheck data (i.e. ingredient data, sales data, employees' data) are saved in the hard disk automatically after each
+ command. There is no need to save manually by entering any command.
+ 
+ 
+<div markdown="span" class="alert alert-primary">
+
+ **:information_source: Notes about data files:**<br>
+User should not attempt to make any change in all data files.<br>
+
+</div>
 
-All tCheck data (i.e. contact details, ingredient data, sales data) are saved in the hard disk automatically after any
- command that changes the data. There is no need to save manually by entering any command.
+#### 4.4 Calendar :
+
+tCheck also shows a current month's calendar when you start the application. You may use it for your working planning.
+
+
+<div markdown="span" class="alert alert-primary">
+
+ **:information_source: Notes:**<br>
+ * In the current version, tCheck can only show the calendar for the current month.<br>
+ 
+ </div>
```

Comment:
The note repeats information in the description. Additionally, it should be "for planning you work" instead.

Reply:
Thanks!